crates/languages/src/json.rs

```diff
@@ -339,11 +342,22 @@ impl LspAdapter for NodeVersionAdapter {
                 archive.unpack(&destination_container_path).await?;
             }
 
-            fs::copy(
-                destination_container_path.join("package-version-server"),
-                &destination_path,
-            )
-            .await?;
+            #[cfg(not(target_os = "windows"))]
+            {
+                fs::copy(
+                    destination_container_path.join("package-version-server"),
```

Comment:
Same comment applies here; let's not cfg things and do a single `format!` call instead:
```suggestion
                    destination_container_path.join(format!("package-version-server{}", std::env::consts::EXE_SUFFIX),
```